app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.tsx

```diff
@@ -105,16 +114,21 @@ function LloydGeorgeUploadPage() {
                         navigate(routes.SESSION_EXPIRED);
                         return;
                     }
+                    await setUploadStateFailed();
                     setStage(LG_UPLOAD_STAGE.FAILED);
                 }
             }
         };
 
         if (hasExceededUploadAttempts) {
             window.clearInterval(intervalTimer);
+            // noinspection JSIgnoredPromiseFromCall
+            setUploadStateFailed();
```

Comment:
```suggestion
            void setUploadStateFailed();
```
For consistency, maybe we could use `void` to mark this promise as not awaited, just like the last "else if" branch in this part.